### DISCUSSION TITLE
743

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#924] [Comments]
# 743

프로그래밍을 배우기 시작한 뒤 부터 노력하지 않는 척을 꽤 자주했다. 이를테면 주말에 사이드 프로젝트나 공부를 했어도 회사에서는 웹툰/영화를 봤던 이야기만 하는 것이다. 주변 사람들에게 저 주말에 공부했어요 라던가, 남는 시간에 사이드 프로젝트나 스터디를 했어요 등을 말하기 싫었다. 당시의 나는 개발을 못했기 때문이다. 노력을 해도 그 정도라면 내가 재능이 없다는 것을 의미하는데 그걸 인정하기 싫었다. 노력을 했음에도 못하는 즉, 재능 없는 사람이 될 바에는 차라리 노력을 안해서 못하는 사람이 되는게 낫다고 생각하게 되었다. 그럼 내가 개발 실력이 부족해도 그건 내 재능이 부족한게 아니라 내 노력이 부족한 것이 되니깐. 내가 선택한 이 분야에서 내가 재능이 없다는 것을 인정하기가 참 어려웠다. 그래서 뭘..

https://jojoldu.tistory.com/743

**Comment by mandarin-sep** [comment]
재능있는척 노력하지 않는 척하면서 안정감을 얻는 거 제가 늘 하던일인데 생각할게 많아지네요... 좋은 글 잘 읽었습니다!

**Comment by sgc109** [comment]
재능이 없다는 사실을 인지하고 꾸준한 노력을 통해 부족함을 메울 수 있는 능력 또한 재능이라고 생각합니다. 좋은글 감사합니다!

**Comment by mag123c** [comment]
노력하면 언젠가는 목표에 근접할 것이라고 끊임없이 달려나가는데 가장 중요한 것은 지치고 낙담하지 않는 것 같아요... 꼭 열심히해서 목표에 도달을!!!!

**Comment by Hoo93** [comment]
100m 17초에 달리기 글과 함께 정말 마음에 큰 도움이 됐습니다.
일하면 일할수록 모자름을 많이 깨닫는 요즘 다시 한 번 꾸준하게 도전해 나갈 수 있는 힘 얻었습니다 ! 
항상 좋은 글 감사합니다 !

**Comment by ParkSeryu** [comment]
학창시절에도 단골멘트였죠..
시험날 물어보면 공부 안했다고, 누가 더 나태했는지 경쟁하기라도 하듯이 말이죠. ㅋㅋ
지금와서 보면 일종의 보험이자 방어기제였다고 생각하는데, 자신의 노력의 결과를 폄하하지 않고 받아들이는게 성장하는 사람의 좋은습관이라는 생각이 듭니다.
좋은 글 감사드려요

**Comment by DoK6n** [comment]
노력이 가장 큰 재능이죠!

**Comment by woohyun1031** [comment]
좋은 글 감사합니다! 정말 순수하게 성찰하게 되네요😂
